frontend/include/chpl/framework/ID.h

```diff
@@ -71,6 +71,12 @@ class ID final {
         numChildIds_(numChildIds) {
   }
 
+  /**
+    Construct an ID with specified symbol path, and default postorder traversal
+    number.
+   */
+  ID(UniqueString symbolPath) : symbolPath_(symbolPath) {}
```

Comment:
I suspect you want to mark this `explicit` to prevent accidental coercion of unique strings to symbol IDs.